modules/storages/app/views/storages/admin/storages/_form.html.erb

```diff
@@ -43,9 +43,13 @@ See COPYRIGHT and LICENSE files for more details.
                    {
                      disabled: @object.persisted? || ::Storages::Storage::PROVIDER_TYPES.count == 1
                    } %>
+      <span class="form--field-instructions"><%= t('storages.instructions.type') %> <%= t('storages.instructions.type_link_text') %></span>
     </div>
     <div class="form--field -required">
-      <%= f.text_field :name, required: true, container_class: '-slim' %>
+      <%= f.text_field :name,
```

Comment:
@HDinger (@apfohl) Well, it actually has a placeholder value (once you delete the value you will see it). However, it was my idea to already put a useful name as a default value. The ideas was that naming the storages by default exactly as the provider type is useful and quick. So I don't see an issue here.

Reply:
The only reason I stumbled over this, is because the ticket explicitly says:
```
'Name' field needs to have a placeholder text, not an actual text
```

Reply:
Yes, that's a fault on my side, I haven't changed the work package accordingly. Is now fixed.

Reply:
Fixed.